rest/changes_api.go

```diff
@@ -225,6 +225,8 @@ func (h *handler) handleChanges() error {
 			return err
 		}
 		feed, options, filter, channelsArray, docIdsArray, _, err = h.readChangesOptionsFromJSON(body)
+		base.LogTo("Changes+", "Changes POST request.  URL: %v, feed: %v, options: %+v, filter: %v, channels: %v, docIds: %v",
+			h.rq.URL, feed, options, filter, channelsArray, docIdsArray)
```

Comment:
Updated, here's how it looks in the logs currently:

```
2017-06-09T16:06:25.328-07:00 Changes+: Changes POST request.  URL: /db/_changes?feed=normal, feed: normal, options: {Since:0 Limit:0 Conflicts:false IncludeDocs:false Wait:false Continuous:false Terminator:<nil> HeartbeatMs:0 TimeoutMs:300000 ActiveOnly:false}, filter: , bychannel: [], docIds: []   (to foo)
2017-06-09T16:06:25.328-07:00 Changes: MultiChangesFeed(channels: {*}, options: {Since:0 Limit:0 Conflicts:false IncludeDocs:false Wait:false Continuous:false Terminator:0xc420367440 HeartbeatMs:0 TimeoutMs:300000 ActiveOnly:false}) ...   (to foo)
```